scripts/sync-to-aws-homebrew-tap

```diff
@@ -178,6 +181,7 @@ cat << EOM > "${BREW_CONFIG_DIR}/${BINARY_BASE}.json"
     "bottle": {
         "root_url": "${BASE_ASSET_URL}",
         "sha256": {
+            "arm64_big_sur": "${MAC_ARM64_HASH}",
```

Comment:
let's just call this one `mac_arm64`. The sierra one is confusing since it's used for all of mac

Reply:
Sure. I was just going with Homebrew's naming conventions.

Reply:
A mostly random example: https://github.com/Homebrew/homebrew-core/blob/master/Formula/awscli.rb